### ISSUE TITLE
Surpress output by default

### ISSUE THREAD

**Issue by ephracis** [KevinBongart/rake-fast#6]
It is very annoying to see the message "Generating .rake_tasks..." when pressing tab, breaking my command. Resulting in the following:

> rake test:ki               
Generating .rake_tasks...
tchen

This should only be printed when debugging, not by default.

**Comment by KevinBongart** [comment]
Yes, that makes sense. I suggest making a PR over at https://github.com/robbyrussell/oh-my-zsh/tree/master/plugins/rake-fast where the plugin is maintained as part of oh-my-zsh
